src/main/resources/docs/description/B105.md

```diff
@@ -1,15 +1,67 @@
-## Hardcoded password string
+**B105: Test for use of hard-coded password strings**
 
-### Rationale:
-The use of hard-coded passwords increases the possibility of password guessing tremendously.
-Variables are considered to look like a password if they have match any one of:
+The use of hard-coded passwords increases the possibility of password
+guessing tremendously. This plugin test looks for all string literals
+and checks the following conditions:
 
-* “password”
-* “pass”
-* “passwd”
-* “pwd”
-* “secret”
-* “token”
-* “secrete”
+-   assigned to a variable that looks like a password
 
-[Reference](https://docs.openstack.org/bandit/latest/plugins/b105_hardcoded_password_string.html)
+-   assigned to a dict key that looks like a password
+
+-   used in a comparison with a variable that looks like a password
+
+Variables are considered to look like a password if they have match any
+one of:
+
+-   “password”
+
+-   “pass”
+
+-   “passwd”
+
+-   “pwd”
+
+-   “secret”
+
+-   “token”
+
+-   “secrete”
+
+Note: this can be noisy and may generate false positives.
+
+**Config Options:**
+
+None
+
+Example
+
+:   
+
+<div class="highlight-none notranslate" markdown="1">
```

Comment:
I think we're losing the Markdown format. Also, there should be no HTML tags here.

Reply:
The last modification should have changed it.